### DISCUSSION TITLE
Is there a way to export chart data to a dataframe

### DISCUSSION THREAD

**Discussion by Jordan-Eckowitz** [ssantoshp/Empyrial#53] [Q&A] [answered]
I'd like to run Empyrial on my backend and serve up the chart data to the frontend of my own personal portfolio app. Is there a way to output the chart data as a dataframe (or similar format)?

**Comment by santoshlite** [comment, marked as answer]
Hey,

First, sorry for the late reply!

Ok, so there are some values you could get. For example, if you do:

```
empyrial.returns = returns  #get dataframe of the returns of your portfolio
empyrial.creturns = creturns  #get dataframe of the cumulative returns of your portfolio
empyrial.benchmark = benchmark  #get dataframe of the returns of your benchmark
```

Now, that you collected these dataframes, you can exploit them to get charts. However, if you want to get the same charts as the one in the report we offer, I advise you to use Quantstats (https://github.com/ranaroussi/quantstats). We use this to get charts from our dataframes. You can get these charts by giving as input the return dataframe.

Good luck with this project!
